scripts/onboarding/aks/onboarding-msi-terraform/main.tf

```diff
@@ -15,6 +15,7 @@ resource "azurerm_kubernetes_cluster" "k8s" {
     name       = "agentpool"
     vm_size    = var.vm_size
     node_count = var.agent_count
+    tags = var.resource_tag_values
```

Comment:
Can you align the = with the above lines

Reply:
@jatakiajanvi12  Thank you for noticing , I have fixed it , please take a look